.github/workflows/release-apps.yml

```diff
@@ -0,0 +1,49 @@
+name: Release apps
```

Comment:
Separate workflow to track changes for `apps/*`. It might present an opportunity to (if we decide to do it) automate Github Release creation.